repository/v.json

```diff
@@ -317,6 +317,20 @@
 				}
 			]
 		},
+		{
+			"name": "View Bookmarks",
+			"details": "https://github.com/adam-zethraeus/SublimeViewBookmarks",
+			"issues": "https://github.com/adam-zethraeus/SublimeViewBookmarks/issues",
+			"readme": "https://raw.githubusercontent.com/adam-zethraeus/SublimeViewBookmarks/master/README.md",
```

Comment:
Since your "details" point to a GitHub repo, you shouldn't add the "issues" and "readme" keys if these things are just in their normal place.